pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -100,13 +100,17 @@ rec {
     nativeVersion = "0.22-milestone-21";
     sha256 = "00h3z0vxc4hv31sc71gb88r8yabyqgz304wpr0bxhbx2d14f11mm";
     defaultJava = jdk17;
+    # Gradle 7 ships some binaires that are only available for some platforms
```

Comment:
```suggestion
    # Gradle 7 ships some binaries that are only available for some platforms.
```

Reply:
I added a link to the supported platforms